bach/bach/series/series_dict.py

```diff
@@ -149,6 +149,14 @@ def elements(self) -> 'DictAccessor':
         """ Accessor to interact with the elements in the dictionary. """
         return DictAccessor(self)
 
+    def materialize(
+            self,
+            node_name='manual_materialize',
+            limit: Any = None,
+            distinct: bool = False,
+    ):
+        raise Exception(f'Series of this type cannot be materialized.')
```

Comment:
I think it's nicer to say the class of the series :) 
```suggestion
        raise Exception(f'{self.__class__.__name__} cannot be materialized.')
```